### ISSUE TITLE
Fix accessibility of anchors in across repo (including README's)

### ISSUE THREAD

**Issue by GeekTrainer** [Microsoft/BotFramework-WebChat#1681]
Several of the links are either "here", or lists of links which are simply labeled "demo". This is not accessible design. Screen readers offer the ability to read a collection of links for easier scanning of a document; in the case of the list of demos all that would be read is "demo", "demo", "demo"...
